src/vue-shim.d.ts

```
@@ -1,5 +1,5 @@
 declare module '*.vue' {
-  import { DefineComponent } from 'vue';
-  const component: DefineComponent<{}, {}, any>;
-  export default component;
+  import {DefineComponent} from 'vue'
+  const component: DefineComponent<{}, {}, any>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)